packaging/installer/kickstart-static64.sh

```diff
@@ -97,10 +100,10 @@ run() {
 
 	local ret=$?
 	if [ ${ret} -ne 0 ]; then
-		run_failed
+		run_failed "${@}"
```

Comment:
Sometimes we use "${*}" as parameter to run_failed, and other times "${@}". Which is correct?